tests/AnonymityTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace geertw\IpAnonymizer\Tests;
+
+use geertw\IpAnonymizer\IpAnonymizer;
+use PHPUnit\Framework\TestCase;
+
+class AnonymityTest extends TestCase
```

Comment:
should probably be named `IpAnonymizerTest`. A common convention is to name the unit test after the class being tested.